Sources/Helpers/TimeIntervalCounter.swift

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 Ordo One AB
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+
```

Comment:
Some description of what this struct is for?
Probably with an example of usage.

Reply:
Added an annotation with description of struct and parameters

Reply:
I agree, I think there should be more decoration with example of usage or use case.

Reply:
Do you mean as a part of documentation or should we use unit tests as example?

Reply:
Btw, should we create a separate case for documentation for `ProcessingRate` as well?